render/render.go

```diff
@@ -14,6 +14,11 @@ import (
 //go:embed template/html.tpl
 var htmlTmpl []byte
 
+type Data struct {
+	Results  types.Results
```

Comment:
Since the graph uses only the Results field from the report, you can change the js logic and pass only the results to the template